lib/core/src/Cardano/Wallet/DB/Sqlite/TH.hs

```diff
@@ -110,7 +111,7 @@ TxMeta
     txMetaAmount            W.Coin              sql=amount
     txMetadata              W.TxMetadata Maybe  sql=data
     txMetaSlotExpires       SlotNo Maybe        sql=slot_expires
-    txMetaFee               Natural Maybe       sql=fee
+    txMetaFee               Word Maybe       sql=fee
```

Comment:
☝️

Reply:
The `Natural` instance were removed with this explanation: https://hackage.haskell.org/package/persistent-2.11.0.2/docs/Database-Persist-Types.html#t:OverflowNatural

I think we might as well use Word instead of OverflowNatural, although maybe we should use `Word64` in case we're not on a 64 bit system?

Reply:
Yep, `Word64` is good, which should map to `SqlInt64`, same as `Natural` did previously.

Reply:
Changed to Word64